gprofiler/utils/__init__.py

```diff
@@ -402,6 +435,9 @@ def atomically_symlink(target: str, link_node: str) -> None:
     """
     tmp_path = link_node + ".tmp"
     os.symlink(target, tmp_path)
+    # Windows does not support renaming existing symlink
```

Comment:
This breaks the "atomically" in this function.

I suggest instead - in the call site of this function, `if is_windows()` then remove the target name with `remote_path(.., missing_ok=True)` and then call `os.symlink`.